extensions/data-workspace/src/main.ts

```diff
@@ -51,6 +52,10 @@ export function activate(context: vscode.ExtensionContext): Promise<IExtension>
 	context.subscriptions.push(vscode.commands.registerCommand('projects.removeProject', async (treeItem: WorkspaceTreeItem) => {
 		await workspaceService.removeProject(vscode.Uri.file(treeItem.element.project.projectFilePath));
 	}));
+	context.subscriptions.push(vscode.commands.registerCommand('projects.manageProject', async (treeItem: WorkspaceTreeItem) => {
+		const dialog = new ProjectDashboard(workspaceService, treeItem);
```

Comment:
copy & paste? the variable name should be dashboard, right? :)

Reply:
Yep, my bad. Fixing it for next iteration.